app/uk/gov/hmrc/thirdpartyapplication/domain/models/ApplicationState.scala

```diff
@@ -80,6 +90,9 @@ object ApplicationState {
   def pendingRequesterVerification(requestedBy: String, verificationCode: String) =
     ApplicationState(State.PENDING_REQUESTER_VERIFICATION, Some(requestedBy), Some(verificationCode))
 
+  def preProduction(requestedBy: String, verificationCode: String) =
```

Comment:
I don't think there is a verification code here.  But oddly I didn't think there was one for production either...